helm/mail/templates/statefulset.yaml

```diff
@@ -116,16 +119,58 @@ spec:
             {{- end }}
             {{- if .Values.extraVolumeMounts }}{{- toYaml .Values.extraVolumeMounts | nindent 12 }}{{ end }}
           resources: {{ toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.metrics.enabled }}
+        - name: exporter
+          imagePullPolicy: Always
+          image: {{ .Values.metrics.image }}
+          ports:
+          - containerPort: {{ .Values.metrics.port }}
+            name: metrics
+            protocol: TCP
+          args:
+          - sleep 15s; /bin/postfix_exporter
```

Comment:
Why `sleep 15s`?

Reply:
The containers complain if the file does not exist. Causing a container failure and restart. I'm sure there is a better way, but I wanted to get it to top yelling at me when I was testing.